packages/mdc-toolbar/component.ts

```diff
@@ -89,5 +96,8 @@ export class MDCToolbar extends MDCComponent {
         }
       },
     });
+    // tslint:enable:object-literal-sort-keys
   }
 }
+
+export default MDCToolbar;
```

Comment:
I actually switched to this split export syntax to make it easier to disable the default export in the future. See #4407 PR description for an explanation.

Reply:
As discussed offline, we're going to remove the `default` exports from `component.ts` and `index.ts` because they do not currently exist in `index.js` on `master`.

Removed.